src/en/status.md

```diff
@@ -60,15 +61,16 @@ If the request is successful, the response body is `json` and the status code is
   "email_address": "sender@something.com",  # required string for emails
   "phone_number": "+447900900123",  # required string for text messages
```

Comment:
We want to get rid of these comments? that says what's required and for which notification types. Do we have other documentation that covers this?

Reply:
I have not removed the comments throughout. Github is highlighting them red but a slightly different shade of red compared to actual removals. Perhaps Github is smart enough to know that the code block is marked as JSON which doesn't support comments?

Reply:
oh alright! sorry about that 😅

Reply:
All good, it caught me off guard the first time I looked at it too!